maths/aliquot_sum.nim

```diff
@@ -0,0 +1,25 @@
+{.push raises: [].}
```

Comment:
maybe add documentation comments with name of algorithm, a brief description (the first sentence from wikipedia are fine) and a link to wikipedia article https://en.wikipedia.org/wiki/Aliquot_sum